src/QGCApplication.cc

```diff
@@ -724,24 +724,6 @@ QObject* QGCApplication::_rootQmlObject()
     return nullptr;
 }
 
-void QGCApplication::showCriticalVehicleMessage(const QString& message)
-{
-    // PreArm messages are handled by Vehicle and shown in Map
-    if (message.startsWith(QStringLiteral("PreArm")) || message.startsWith(QStringLiteral("preflight"), Qt::CaseInsensitive)) {
-        return;
-    }
-    QObject* rootQmlObject = _rootQmlObject();
-    if (rootQmlObject) {
-        QVariant varReturn;
-        QVariant varMessage = QVariant::fromValue(message);
-        QMetaObject::invokeMethod(_rootQmlObject(), "showCriticalVehicleMessage", Q_RETURN_ARG(QVariant, varReturn), Q_ARG(QVariant, varMessage));
-    } else if (runningUnitTests()) {
```

Comment:
This might mean that tests might fail (if we ran them), are there tests somewhere?

Reply:
Yes, we should start adding unit tests for features when we can

Reply:
There are no existing tests for vehicle.cc or UASMessageHandler.cc, so I think I will need some help creating unit tests for this functionality